### ISSUE TITLE
Plain functions as helpers fail with hash params due to test in ember-could-get-used-to-this

### ISSUE THREAD

**Issue by johanrd** [nickschot/ember-mobile-menu#771]
1. Clone this repo: https://github.com/johanrd/plain-functions-as-helpers-with-hash-params

See that the repo fails with after installing ember-mobile-menu:
```
Error: Assertion Failed: Functional helpers cannot receive hash parameters. `substring` received start,end
```
this is the [most plain substring function from the official ember guides](https://guides.emberjs.com/release/components/helper-functions/#toc_named-arguments-1 ).

The error message itself stems from [ember-could-get-used-to-this](https://github.com/tracked-tools/ember-could-get-used-to-this/blob/10298d192e42ee3a87f654344a368ddb5c4dffce/tests/integration/functions-test.js#L82), where plain functions as helpers (both positional and named args) are included in ember > 4.5.

Not sure if this should be posted in `ember-mobile-menu` or `ember-could-get-used-to-this`, but it seems like the test can be removed if/when used by modern ember apps?

**Comment by nickschot** [comment]
Thanks for looking into this, it was indeed my plan to replace ember-could-get-used-to-this with the more modern alternative ember-resources. I'll review your PR asap.

**Comment by nickschot** [comment]
Released in pre-release [5.0.0-1](https://github.com/nickschot/ember-mobile-menu/releases/tag/v5.0.0-1) 👍
